metabrainz/templates/index/gdpr.html

```diff
@@ -64,7 +64,7 @@ <h2>{{ _('Right to be forgotten') }}</h2>
        <a href="https://github.com/metabrainz/musicbrainz-server/blob/master/lib/MusicBrainz/Server/Data/Editor.pm">How we delete 
        accounts in MusicBrainz</a> (warning, geek content ahead!)
      </li>
-     <li>Implement site wide user account deletion in 
+     <li>Implement site-wide user account deletion in 
```

Comment:
Should this actually be "sitewide"? ([Wiktionary](https://en.wiktionary.org/wiki/sitewide))